web/impact/impact/v1/serializers/office_hours_serializer.py

```diff
@@ -30,13 +31,27 @@ class Meta:
             'topics', 'description', 'location',
         ]
 
+    def handle_conflicting_sessions(self, attrs):
```

Comment:
the def of done says
> If a create or edit request would create any office hour session in conflict with any existing session for the current user, return an error



This only checks one session at a time, so if the first session(s) requested are not in conflict, they are created, which is incorrect.

Reply:
That is not the case, `handle_conflicting_sessions` will be called on every session and if any of the session has an error, non will be created [office_hour_view.py#L122-L125](https://github.com/masschallenge/impact-api/blob/6160eded8d31962dd6e52dc7f23ba36753efd1ef/web/impact/impact/v1/views/office_hour_view.py#L122-L125)

Reply:
👍